devicetypes/smartthings/zigbee-lock-without-codes.src/zigbee-lock-without-codes.groovy

```diff
@@ -212,6 +214,12 @@ private def parseAttributeResponse(String description) {
 	return result
 }
 
+private def parseIasMessage(String description) {
+	ZoneStatus zs = zigbee.parseZoneStatus(description)
+	def responseMap = [ name: "battery", value: zs.isBatterySet() ? 5 : 50, isStateChange: true]
```

Comment:
why are we forcing `isStateChange`?

Reply:
to see that I got event about battery state

Reply:
But we wouldn't want it typically. Filling the event logs with battery 50% doesn't fix anything, does it?

Reply:
I accept your argument. Let's go ahead and merge PR.